README.md

```diff
@@ -11,86 +11,131 @@
 </p>
 
 # Fivetran Salesforce Formula Utils
-
+# 📣 What does this dbt package do?
 This package includes macros to be used within a Salesforce dbt project to accurately map Salesforce Formulas to existing tables.
```

Comment:
I would perhaps include some links to the salesforce ERD and connector to be in line with some of our other READMEs, for ex google_ads https://github.com/fivetran/dbt_google_ads_source#-what-does-this-dbt-package-do